molecule/cookiecutter/molecule/{{cookiecutter.role_name}}/.yamllint

```diff
@@ -7,5 +9,25 @@ rules:
   brackets:
     max-spaces-inside: 1
     level: error
+  colons:
+    max-spaces-after: -1
+    level: error
+  commas:
+    max-spaces-after: -1
+    level: error
+  comments: disable
+  comments-indentation: disable
+  document-start: disable
+  empty-lines:
+    max: 3
+    level: error
+  hyphens:
+    level: error
+  indentation: disable
```

Comment:
Late to the party, but @ericsysmin - I strongly question why we would want to disable indentation checking by default? `yamllint` has it active by default so this definitely changes the behavior from what we had previously.

I understand the desire to harmonize with galaxy lint settings though. In this case, it would perhaps be better to leave questionable rules like this commented out with a clear comment indicating that they can be enabled if the use wishes to harmonize with galaxy linter settings.

Likewise with `new-line-at-end-of-file` and `trailing-spaces`; it feels odd to disable these rules.

Reply:
An issue / PR with change may attract more attention to this.

Reply:
No need to reopen this discussion because we are very close of removing the role template completely from molecule and rely on galaxy. Still, I support @perlun idea that yamllint is useful. I only advise to take this to ansible-galaxy and fix it there. I will be glad to support the change.